tortoise.py

```diff
@@ -0,0 +1,11 @@
+TORTOISE_FILE = 'tiny.tortoise'
+
+def main():
+    with open(TORTOISE_FILE, 'r') as tortoise_commands:
+        tortoise_program = tortoise_commands.readlines()
+        tortoise_commands.close()
```

Comment:
I'm not going to tell you what's wrong with this. Instead, I'm going to tell you to read up on `with` blocks and context managers. Let me know if you get stuck.